site_suitability_input_conditions_form_v16_20240305_with_example_data.json

```diff
@@ -251,7 +265,7 @@
             "Associated Data Source": "Gobblers Knob East",
             "Ve50": 59.36000000000001,
             "V50": 42.400000000000006,
-            "CoV": "N/A",
+            "COV": "N/A",
```

Comment:
is this correct?  I think "CoV" had a specific scientific meaning.  @NievesBurillo ? @youngli-jp

Reply:
#10 linking the related issue here for resolution